miyagi_trainer/train.py

```diff
@@ -149,6 +149,7 @@ def train_model(model,
             if track_experiment:
                 if phase == "val" and epoch_acc > best_acc:
                     wandb.run.summary["best_val_acc"] = epoch_acc
+                    best_acc = epoch_acc
```

Comment:
That doesn't appear to be right, because we want to maintain the option for the user to save the best model (which uses the `best_acc`/`best_eer` and to *not* track the experiment. But maybe I'm missing something.

I guess the best way is to update outside the `if` at 149.

Reply:
Sure, I agree.